tests/providers/test_run_job.json

```diff
@@ -0,0 +1,17 @@
+{
+	"version": "1",
+	"date_created": "2024-01-01T00:00:00+0000",
+	"date_updated": null,
+	"steps":
+	[
+		{
+			"action": "process",
+			"args": {
+				"source_paths": [".assets/examples/source.jpg"],
+				"target_path": ".assets/examples/target-240p.jpg",
+				"output_path": ".assets/examples/test_swap_face_to_image.jpg"
```

Comment:
this is just a example, but most of this files override existing ones - it should be test_job_{name}